www/javascript/site-content-slider.js

```diff
@@ -66,6 +68,8 @@ YAHOO.util.Event.onDOMReady(function ()
 		this.touch_start_y = null;
 		this.touch_end_x = null;
 
+		this.pageChangeEvent = new YAHOO.util.CustomEvent('pageChange', this);
```

Comment:
Does `pageChange` fire on browser resizing? If so, should this be [debounced](http://davidwalsh.name/javascript-debounce-function)?